courses/machine_learning/production_ml_systems/1_training_at_scale.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<p>When I ran this I got an error initially and had to run <code>gcloud auth configure-docker</code> to get it to work.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/584/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Added! Thanks for spotting that!</p>